packages/eden/src/links/internal/observable.ts

```diff
@@ -178,6 +178,7 @@ export class Observable<TValue = any, TError = any> extends Subscribable<TValue,
   ): Observable<TValue5, TError5>
 
   pipe(...operations: OperatorFunction[]): Observable {
-    return operations.reduce(pipeReducer, this)
+    // eslint-disable-next-line unicorn/no-array-reduce
+    return operations.reduce((accumulator, element) => pipeReducer(accumulator, element), this)
```

Comment:
What's the point of explicitly passing in the arguments?
```suggestion
    return operations.reduce(pipeReducer, this)
```